platform_versioned_docs/version-24.2/compute-envs/google-cloud-batch.mdx

```diff
@@ -190,7 +190,11 @@ Wave containers and Fusion v2 are recommended features for added capability and
 
 #### GCP resources
 
-Enable **Spot** to use Spot instances, which have significantly reduced cost compared to on-demand instances.
+Enable **Spot** to use Spot Instances, which have significantly reduced cost compared to On-Demand Instances. 
```

Comment:
![image](https://github.com/user-attachments/assets/b3fbc349-5109-4506-ac80-83e048983469)
"Instance" is not capitalized in the GCP docs, so it shouldn't be capitalized in ours.